### ISSUE TITLE
PipeWire multi-arch cross-build Discussion

### ISSUE THREAD

**Issue by pipewire-debian** [pipewire-debian/pipewire-debian#67]
See #64 
@tormodvolden

**Comment by tormodvolden** [comment]
If you're limited on time, I'd suggest focusing on amd64. A reasonable goal is to provide the latest PipeWire and associated libraries and tools for a large population of eager testers, and this can also help upstream to get better test coverage across usage patterns and equipment. More exotic architectures are left to upgrade their system e.g. Debian unstable.

Another policy would be to do like Debian, disable certain features for some architectures. This means you're spending minimal time on these other architectures, and many architectures are supported almost at no extra effort. Seems like a good compromise to me.

Trying to keep everything consistent across all architectures would require a lot of upstream work.

**Comment by pipewire-debian** [comment]
> Another policy would be to do like Debian, disable certain features for some architectures. This means you're spending minimal time on these other architectures, and many architectures are supported almost at no extra effort. Seems like a good compromise to me.

Yeah, I also thought like this. Its better to have a compromised pipewire other than nothing for other archs. As you can see as of now our PPA is ready to accept other architecture we can do for pipewire also with trimmed features.

Yeah have less time, but you can even give some MR for this :sweat_smile:  otherwise it will take time for me.

**Comment by pipewire-debian** [comment]
Hi @tormodvolden  
Can you please review 750e4a7cc428afd0bfa872490be95c94f678578d?

**Comment by DanceMore** [comment]
I landed here because I needed extended Pipewire tooling on **Debian**, like in the repo name...

Apparently, **Ubuntu** dpkg supports a compression algorithm that Debian dpkg does not => https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=892664

I get  `"unknown compression for member 'control.tar.zst'"` when trying to use the latest `blueman-git` package from the PPA on Debian Unstable.

Maybe while adjusting for multiple architectures, you could also downgrade the compression so the packages are actually installable on **Debian** ?

**Comment by souravdas142** [comment]
@DanceMore  which version of ubuntu are you using for debain unstable ?

for only blueman you can use https://github.com/blueman-project/blueman/wiki/Packaged-versions for now. 

I will fix this in next version https://stackoverflow.com/a/72359798
